### ISSUE TITLE
Why did I get better results than the paper

### ISSUE THREAD

**Issue by alexanderuo** [HRNet/HRNet-Semantic-Segmentation#253]
![508E_` V`{29CY41BQ62~R](https://user-images.githubusercontent.com/36657510/148359038-34f4582d-3347-4709-9401-55c40f9a363c.png)
Pre-training model ：hrnet_ocr_cs_trainval_8227_torch11.pth
Maybe because of a previous error：The file could not be found ：final_state.pth 
I changed the name of the hrnet_ocr_cs_trainval_8227_torch11.pth model to this one and put it in the corresponding path. Meanwhile, I would like to ask how to get final_state.pth ,Maybe there's something wrong here

**Comment by ydhongHIT** [comment]
> ![508E_ V{29CY41BQ62~R](https://user-images.githubusercontent.com/36657510/148359038-34f4582d-3347-4709-9401-55c40f9a363c.png) Pre-training model ：hrnet_ocr_cs_trainval_8227_torch11.pth Maybe because of a previous error：The file could not be found ：final_state.pth I changed the name of the hrnet_ocr_cs_trainval_8227_torch11.pth model to this one and put it in the corresponding path. Meanwhile, I would like to ask how to get final_state.pth ,Maybe there's something wrong here

Hi, I think that the trainval model was trained on both train and val set. Thus, the eval result on the val set is high. You should test it on the test set or evaluate the models trained on only train set.
